Tasks/JavaToolInstallerV0/task.json

```diff
@@ -13,8 +13,8 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 0,
-        "Minor": 211,
-        "Patch": 0
+        "Minor": 216,
+        "Patch": 4
```

Comment:
Please don't forget to change Patch version to "0"

Reply:
fixed